changedetectionio/content_fetcher.py

```diff
@@ -653,11 +656,11 @@ def run(self,
     # Does the connection to the webdriver work? run a test connection.
     def is_ready(self):
         from selenium import webdriver
-        from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+        from selenium.webdriver.chrome.options import Options as ChromeOptions
 
         self.driver = webdriver.Remote(
             command_executor=self.command_executor,
-            desired_capabilities=DesiredCapabilities.CHROME)
+            options=ChromeOptions())
```

Comment:
last question i think - if someone runs this code but with the older selenium, what will happen? does it work still?

Reply:
If it were me I would probably mark it as a potentially breaking change in the release notes to be safe, but I will let @marceloalencar answer

Reply:
That syntax was introduced in version 3.8.0. It should still be compatible, since the lowest required version for changedetection was 4.1.0.

Reply:
selenium 4.14.0 is still able to drive the older `selenium/standalone-chrome-debug:3.141.59` container, if the user chooses to not update

Reply:
@marceloalencar fantastic, thank you , sorry for the delay btw